js/components/closed-caption-multi-audio-menu/tab.js

```diff
@@ -19,24 +22,36 @@ var Tab = React.createClass({
           speed={1}
           horizontal={false}
         >
-          <ul className="oo-cc-ma-menu__list">
+          <ul
+            className="oo-cc-ma-menu__list"
+            role={CONSTANTS.ARIA_ROLES.MENU}
+          >
             {this.props.itemsList.map(function(el, index) {
               return (
                 <li
                   key={index}
-                  onClick={this.handleClick.bind(this, el.id)}
                   className={classnames('oo-cc-ma-menu__element', {
```

Comment:
I recommend adding a `role="presentation"` attribute to the `li` element, otherwise NVDA will not detect the correct number of items in this menu.

Reply:
\+